src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.CustomScript
+{
+    public class CustomScript : NotificationBase<CustomScriptSettings>
```

Comment:
I had external script originally, I felt it was a bit awkward, but I'd be okay with it. @Taloth whats your opinion?